### PR TITLE
[11.0][ADD] base_address_city: Nothing to do

### PR DESCRIPTION
**Pull request by yuriqp**
- Updated documentation
- Analisis work file

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/openupgrade/doc/source/modules100-110.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ missing in the new release are marked with |del|.
 +-----------------------------------+-----------------------------------+
 | |del| base_action_rule            |                                   |
 +-----------------------------------+-----------------------------------+
-| |new| base_address_city           |                                   |
+| |new| base_address_city           |    Done                           |
```
Comment (StefanRijnhart) [inline odoo/openupgrade/doc/source/modules100-110.rst]: Please change to `Nothing to do`

**Comment by StefanRijnhart** [comment]
Thanks! I think we need @jholze to add you to the Bloopark CLA.

**Review by lepistone — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@yuriqp please sign the CLA for being able to merge this.

**Comment by StefanRijnhart** [comment]
@pedrobaeza there is no code, so I'm suggesting to merge this without CLA. What do you think?

**Comment by pedrobaeza** [comment]
Well, the problem is about the commit itself. In case of any automatic extraction of authors in case of issue, it will appear and we will need to manually filter that.

**Comment by yuriqp** [comment]
I already signed the CLA, should I do something else?

**Comment by pedrobaeza** [comment]
Did you send it to cla@odoo-community.org? Last time I checked it was not there.

**Comment by yuriqp** [comment]
Yes. I did on october 22. I received a confirmation on october 27.

**Comment by pedrobaeza** [comment]
OK, let me check again. Sorry for bothering you with this bureaucracy issues.

**Comment by yuriqp** [comment]
No problem

**Comment by pedrobaeza** [comment]
CLA checked. Everything is OK. Thanks. Merging...

**Pull request merged**